shared/src/main/scala/squants/thermal/Temperature.scala

```diff
@@ -9,6 +9,7 @@
 package squants.thermal
 
 import squants._
+import squants.Platform.crossFormat
```

Comment:
Interesting, is this really the only place this is being used? Regardless I like this solution